src/main/java/com/softserve/messaging/controller/MessageHandlerController.java

```diff
@@ -0,0 +1,32 @@
+package com.softserve.messaging.controller;
+
+import com.softserve.messaging.model.MessageHandler;
+import com.softserve.messaging.repository.MessageHandlerRepo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/messaging/mailing")
+public class MessageHandlerController {
+
+    private final MessageHandlerRepo messageHandlerRepo;
+
+    @GetMapping("/{mail}/receipts")
```

Comment:
* id of the resource should be its primary key. As I see, it's not an email
* path is usually logged, so it's safer to pass email as a RequestParam